src/core/ref.jl

```diff
@@ -1,16 +1,18 @@
-""
-function calc_equal_repairs_per_period(pm::_PMs.GenericPowerModel; nw::Int=pm.cnw)
-
+function calc_cumulative_repairs_per_period(pm::_PMs.GenericPowerModel; nw::Int=pm.cnw)
```

Comment:
By computing this value in the data model and again here, we run the risk of the two quantities getting out of sync.  If we simply store the value of `cumulative_repairs` and then look it up when posting  `constraint_restoration_cardinality` then  this function can be eliminated entirely.

Reply:
Done